src/app/Shared/Services/Target.service.tsx

```diff
@@ -36,19 +36,19 @@
  * SOFTWARE.
  */
 import { Observable, Subject, BehaviorSubject } from 'rxjs';
+import { Target } from './Services';
 
 class TargetService {
-
-  private readonly _target: Subject<string> = new BehaviorSubject('');
+  private readonly _target: Subject<Target> = new BehaviorSubject({} as Target);
```

Comment:
I think it's worth defining some specific publicly-readable constant value as a `NO_TARGET`, which could be defined as the empty object `{}` or something else. But this way we have a consistent value to use to represent this case.

Reply:
Would this be in Target.service.tsx and then have to imported anywhere its needed?

Reply:
Yes, that sounds about right. It could also be a member of the `context.target` instance itself, but that doesn't seem quite right.